File: Easy/PYTHON/NthFibonacci.py
def get_nth_fibonacci(n):
    if n <= 1:
        return 0
    elif n ==2:
        return 1
    else:
        return get_nth_fibonacci(n-1) + get_nth_fibonacci(n-2)


result = get_nth_fibonacci(6)
print(result)

File: Easy/PHP/NthFibonacci.php
<?php

namespace Easy\PHP\NthFibonacci;

function getNthFibonacci($n)
{
    if ($n <= 1) {
        return 0;
    } elseif ($n == 2) {
        return 1;
    } else {
        return getNthFibonacci($n - 1) + getNthFibonacci($n -2);
    }
}

$result = getNthFibonacci(6);
print_r($result);
